app/src/main/java/eu/darken/capod/main/ui/overview/OverviewFragment.kt

```diff
@@ -147,7 +149,19 @@ class OverviewFragment : Fragment3(R.layout.main_fragment) {
         vm.launchUpgradeFlow.observe2 {
             it(requireActivity())
         }
+
         super.onViewCreated(view, savedInstanceState)
+
+        ViewCompat.setOnApplyWindowInsetsListener(view) { v, insets ->
+            val systemWindowInsets = insets.getInsets(WindowInsetsCompat.Type.systemBars())
+            v.setPadding(
+                v.paddingLeft,
+                systemWindowInsets.top,
+                v.paddingRight,
+                0
+            )
+            insets
+        }
```

Comment:
Could we add this to `Fragment2` and have it applied to all fragments in the app?

Reply:
Yep :)
That seemed to work just fine, so its been moved in later commits